static/stable/stage/modules/fed-modules.json

```diff
@@ -1244,5 +1244,22 @@
   "runtimes": {
       "manifestLocation": "/apps/runtimes/fed-mods.json",
       "modules": []
+    },
+   "roadmap": {
+       "manifestLocation": "/apps/roadmap/fed-mods.json",
+       "modules": [
+           {
+               "id": "roadmap",
+               "module": "./RootApp",
+               "routes": [
```

Comment:
@beav the issue we were seeing with the duplicate `/roadmap/roadmap` in the URL is due to these routes. Can we update in a follow up PR to:
```json
"roadmap": {
       "manifestLocation": "/apps/roadmap/fed-mods.json",
       "modules": [
           {
               "id": "roadmap",
               "module": "./RootApp",
               "routes": [
                {
                    "pathname": "/insights/roadmap"
                }
	       ]
	   }
	]
  }
```